src/pages/index.js

```diff
@@ -45,30 +73,68 @@ const section = new Section(
 );
 section.renderItems();
```

Comment:
`section.renderItems();` is not needed because you should render the items only when you get them from the server in line 210